app/Notifications/NewRequestClaim.php

```diff
@@ -42,6 +43,24 @@ public function via(object $notifiable): array
         return ['database'];
     }
 
+    /**
+     * Determine if the notification should be sent.
+     */
+    public function shouldSend(User $notifiable): bool
+    {
+        if ($notifiable->notification?->block_notifications == 1) {
```

Comment:
Should probably not notify the requester if the claimer is the same user

Reply:
Aded self check.